src/styles/Admin.module.css

```diff
@@ -0,0 +1,7 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
```

Comment:
I would remove this height constraint, or set it to use different units. Working with `vh` can get pretty wonky so I'd rather not use it if we don't have to.

I don't think the container needs any height styling. If you remove the height property and find yourself having issues with container height you can add some styles to `global.css` and `_app.tsx`.

**NOTE**: don't spend too much time on this, it's a minor nitpick. If you spend more than 15 minutes on this just re-submit the PR and I'll clean it up.